usr/share/rear/finalize/Linux-ppc64le/620_install_grub2.sh

```diff
@@ -40,15 +40,24 @@ if [[ -r "$LAYOUT_FILE" ]]; then
     LogIfError "Unable to find /boot/$grub_name/grub.cfg."
 
     # Find PPC PReP Boot partition
-    part=$( awk -F ' ' '/^part / {if ($6 ~ /prep/) {print $7}}' $LAYOUT_FILE )
-
-    if [ -n "$part" ]; then
-        LogPrint "Boot partition found: $part"
-        dd if=/dev/zero of=$part
-        # Run grub-install/grub2-install directly in chroot without a login shell in between, see https://github.com/rear/rear/issues/862
-        chroot $TARGET_FS_ROOT $grub_name-install $part
-        NOBOOTLOADER=
-    fi
+    part_list=$( awk -F ' ' '/^part / {if ($6 ~ /prep/) {print $7}}' $LAYOUT_FILE )
+
+    # If software RAID1 is used, several boot device will be found
+    # need to install grub2 on each of them
+    for part in $part_list ; do
+        if [ -n "$part" ]; then
+            LogPrint "Boot partition found: $part"
+            dd if=/dev/zero of=$part
+            # Run grub-install/grub2-install directly in chroot without a login shell in between, see https://github.com/rear/rear/issues/862
+            # When software RAID1 is used, grub2 needs correct PATH to access other tools
+            if chroot $TARGET_FS_ROOT /bin/bash -c "PATH=/sbin:/usr/sbin:/usr/bin:/bin $grub_name-install $part" ; then
```

Comment:
Do you need anything else from bash here? Otherwise it might be better to use `env` like this:
```bash
chroot $TARGET_FS_ROOT /usr/bin/env PATH=/sbin:/usr/sbin:/usr/bin:/bin $grub_name-install $part
```